### DISCUSSION TITLE
tutorials/voyage-part1/01-intro-to-federation

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#87] [course feedback]
# tutorials/voyage-part1/01-intro-to-federation

Learn the basics of federation and the benefits of modularizing your graph.

https://www.apollographql.com/tutorials/voyage-part1/01-intro-to-federation

**Comment by luhtaf** [comment]
thanks for the explanation, i already imagine the apps will be like many microservice and be manage by API gateway

**Comment by diego11404** [comment]
muy detallado esta primera parte
